posterior_database/models/stan/ratings_2_model.stan

```diff
@@ -0,0 +1,16 @@
+data {
+  int N;
+  vector[N] y;
+  int J;
+  int<lower=1, upper=J> movie[N];
+}
+parameters {
+  vector<lower=0, upper=5>[J] theta;
+  real<lower=0, upper=2.5> sigma_y;
+}
+model {
```

Comment:
What is the difference of this model vs the model 1?

Reply:
This is the only source for the 'ratings' models I can find:
https://github.com/stan-dev/example-models/blob/master/knitr/ratings/story.Rmd

They are used to analyse a data set of simulated movie ratings. In the text, it introduces the model by only accepting 2 movies (ratings_1) which is hard-coded in ratings_1_model. 
The model is afterwards extended to 'J' models, so ratings_2_model has an extra piece of data 'J' that is used instead of 2. 
I guess ratings_2_model could be used for both by adding extra J variable in ratings 1 data, but they are separately done in the source. The example-models does not include pre-generated data so I made my own in this case.